config/software/libiconv.rb

```diff
@@ -42,6 +42,11 @@
     patch source: "v1.14.ppc64le-ldemulation.patch", plevel: 1
   end
 
+  # Update config.guess to support newer platforms (like aarch64)
+  if version == "1.14"
+    patch source: "config.guess_2015-09-14.patch", plevel: 0
+  end
```

Comment:
This patch conflicts with ``v1.14.ppc64le-configure.patch`` above. Removing that patch seems to work.